docs/src/pages/landing-page/Index.vue

```diff
@@ -207,14 +209,23 @@ import { defineComponent } from 'vue'
 import WhyQuasarCard from 'src/components/landing-page/WhyQuasarCard.vue'
 import { sponsorLogos, whyQuasar } from 'src/assets/landing-page/image-links.js'
 import TwitterShowcaseCards from 'src/components/landing-page/TwitterShowcaseCards.vue'
+import { socialLinks } from 'assets/landing-page/social-links.js'
 
 export default defineComponent({
   name: 'Index',
   components: { TwitterShowcaseCards, WhyQuasarCard },
   setup () {
+    const footerSocialLinks = socialLinks.reverse().slice(0, 4)
+
+    function scrollSectionIntoView (idOfTarget) {
+      document.getElementById(idOfTarget).scrollSectionIntoView({ behavior: 'smooth' })
```

Comment:
Check out Quasar scroll utils.
They're a bit less easy to use than native method, but works cross browser.
Eg. Safari doesn't support scroll behaviour